### PR TITLE
Example 3

### PR DESCRIPTION
**Pull request by el-angel**

### CODE REVIEW

File: example-3/style.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
+body {
```
Comment (Marinolinderhof) [inline example-3/style.css]: filename is a mismatch  should be styles

File: example-1/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Examples 1
+
+For more or other examples please check out our main [README](../)
+
+This example shows
+
+- Floating labels
```
Comment (Marinolinderhof) [inline example-1/README.md]: example one doesn't have floating labels

**Comment by Marinolinderhof** [comment]
i miss the README in this example

File: example-2/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+For more or other examples please check out our main [README](../)
+
+This example showcases
+
+- Dark layout (supply options for styling to Mollie Component inputs)
+- Native CSS styling
+- ES5 Compatible
+
+[HTML](example-2/index.html) | [CSS](example-2/style.css) | [Javascript](example-2/script.js)
```
Comment (pabloroman) [inline example-2/README.md]: These links are broken:

```suggestion
[HTML](./index.html) | [CSS](./style.css) | [Javascript](./script.js)
```

File: example-3/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+For more or other examples please check out our main [README](../)
+
+This example showcases
+
+- Round inputs
+- Native CSS styling
+- ES5 Compatible
+
+[HTML](example-3/index.html) | [CSS](example-3/style.css) | [Javascript](example-3/script.js)
```
Comment (pabloroman) [inline example-3/README.md]: These links are broken:

```suggestion
[HTML](./index.html) | [CSS](./styles.css) | [Javascript](./script.js)
```

File: example-3/README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+![](screenshots/example-3.gif)
```
Comment (pabloroman) [inline example-3/README.md:13]: There appears to be no screenshots. You can add one or remove this line instead

File: example-2/README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+![](screenshots/example-2.gif)
```
Comment (pabloroman) [inline example-2/README.md:13]: There appears to be no screenshots. You can add one or remove this line instead

**Review by pabloroman — CHANGES_REQUESTED** [review]
This example has a colored background which is not matching the input when I try to run it in codesandbox, can you take a look at why does this happen? This only happens when using the autocomplete feature in my browser

https://codesandbox.io/s/thirsty-hofstadter-172o9

image

image

File: example-3/README.md (hunk 5 above)
Comment (el-angel) [reply example-3/README.md:13]: Screenshots will be added later, we'll ask Jaap.

**Comment by el-angel** [comment]
> This example has a colored background which is not matching the input when I try to run it in codesandbox, can you take a look at why does this happen? This only happens when using the autocomplete feature in my browser
> 
> https://codesandbox.io/s/thirsty-hofstadter-172o9
> 
> image
> 
> image

this is due to browser implementation that cannot be changed. We can only, with css, change the background color of that input (the part which is white in your screenshot)

**Review by Marinolinderhof — APPROVED** [review]

**Pull request merged**